db/model/room.js

```diff
@@ -33,6 +34,14 @@ module.exports = function room(seq, dataTypes) {
       },
       comment: 'Create a named room ',
     },
+    settings: {
+      type: dataTypes.ARRAY(dataTypes.JSONB),
+      allowNull: true,
+      validate: {
+        contains: u.validateSettingsArray,
+      },
+      comment: 'Key/Value pairs for user specific settings',
```

Comment:
I think we decided on the array of objects because its an unordered list. As brandon mentioned this design has been implemented on room types already and the room is inheriting it.